lib/embedded_ansible.rb

```diff
@@ -295,3 +303,5 @@ def self.setup_complete_file
   end
   private_class_method :setup_complete_file
 end
+
+Dir.glob(File.join(File.dirname(__FILE__), "embedded_ansible/*.rb")).each { |f| require_dependency f }
```

Comment:
I think that these shouldn't actually be autoloaded because the platform should be detected by the `EmbeddedAnsible.new` method.

The discussion was prompted because I was having trouble with the specs, but I solved that with `require_dependency 'embedded_ansible'`